main.go

```diff
@@ -66,7 +66,7 @@ func discordMainWrapper() error {
 		shardID = -1
 	}
 
-	port := os.Getenv("SERVER_PORT")
+	port := os.Getenv("PORT")
 	num, err := strconv.Atoi(port)
 	if err != nil || num < 1024 || num > 65535 {
 		log.Printf("Invalid or no particular SERVER_PORT (range [1024-65535]) provided. Defaulting to %s\n", DefaultPort)
```

Comment:
Mind changing this debug test as well, if we're changing to PORT instead of SERVER_PORT? I do agree with the change though, not sure why I chose the verbose SERVER_PORT...

Reply:
Oh yea, I missed the log statement, I'll do that.